packages/flutter_tools/lib/src/base/build.dart

```diff
@@ -311,7 +311,7 @@ class AOTSnapshotter {
       TargetPlatform.android_arm64,
       TargetPlatform.android_x64,
       TargetPlatform.ios,
-      TargetPlatform.darwin_x64,
+      TargetPlatform.darwin,
```

Comment:
I agree with you in https://github.com/flutter/flutter/issues/81205#issuecomment-827131666
> It does feel a bit off that there is `TargetPlatform.ios` and `--define=IosArchs=...` and then `TargetPlatform.darwin`. But iOS _is_ darwin. It really should be `TargetPlatform.macOS` and `--define=MacOSArchs=...`.

Just glancing at this I wouldn't be sure which to use (the confusion predates this PR).  This also gets hairier when we start targeting iOS apps as native ARM mac apps for the unfortunately named "Mac Designed for iPad" https://github.com/flutter/flutter/issues/66184.

I'd be supportive if you wanted to rename this `macos` instead of `darwin`.

Reply:
> I'd be supportive if you wanted to rename this macos instead of darwin.

I'll keep that in mind, but I'm not sure I'd want to do it in this PR.